modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/detail-view-table/index.html

```diff
@@ -11,15 +11,15 @@
 	</thead>
 
 	<tbody class="ml-1">
-		[#list 0..configuration.numberOfRows-1 as i]
+		[#list 0..configuration.numberOfRows as index]
```

Comment:
[#list 1..configuration.numberOfRows as index]
as we are removing the -1 the count of the list has to start at 1